mbed_devices/mbed_tools/cli.py

```diff
@@ -67,3 +70,11 @@ def _build_json_output(devices: Iterable[Device]) -> str:
             }
         )
     return json.dumps(devices_data, indent=4)
+
+
+def _get_build_targets_str(mbed_target: MbedTarget, delimiter: str) -> str:
```

Comment:
Maybe this should return a list of variants instead. It'd be nice if the knowledge of build variants was encapsulated in one place instead of doing string concats with `board_type` elsewhere.
I.e.: `def _get_build_targets(device: Device) -> Iterable[str]`, then each _presentation layer_ can do with it whatever it needs.

Reply:
Already changed in response to the JSON comment